docs/postgresql-server.md

```diff
@@ -5,10 +5,8 @@ Percona Server for PostgreSQL is a binary-compatible, drop-in replacement for Po
 * Storage Manager (SMGR) API Exposure: Allows PostgreSQL extensions to integrate custom storage managers. This change was inspired by the [patchset](https://www.postgresql.org/message-id/flat/CAJ7c6TOqqrzjYsU6LgDkcJ0yVgzdkx2juJjgAjzP2jPOpZ1qUA%40mail.gmail.com#8e68cfc57fcac14c8e24b00b41e61baf) introduced to the community.
 * WAL Read/Write API Exposure to hook into WAL read and write functions.
 
-These modifications provide index-level and WAL encryption of indexes via the `pg_tde` extension. Note that these encryption features are experimental and under active development.
+These modifications are the preparations to provide index-level and WAL encryption of indexes via the [`pg_tde` :octicons-link-external-16:](https://percona-lab.github.io/pg_tde/main/) extension. These encryption features are still under active development and are planned for future releases.
```

Comment:
Can we rephrase to say that these changes are not impacting on existing use cases unless a specific pg_tde extension build is used? 

These changes can potentially be only used by opt-in to a tech preview of such a build.

Phrasing it in the current fashion suggests that the patch is under active development while the reality is that the extension is